src/stdatamodels/jwst/datamodels/model_base.py

```diff
@@ -55,7 +55,6 @@ def on_save(self, init):
         self.meta.date = Time.now().isot
 
 
-# We may want to deprecate DataModel
-# @deprecate_class(JwstDataModel)
 class DataModel(JwstDataModel):
-    pass
+    def __init_subclass__(self):
+        warnings.warn("Class DataModel has been renamed JwstDataModel. It will be removed in v 1.5.", DeprecationWarning, 2)
```

Comment:
Previously this class had no `__init__` function. If I add a call to `super` some tests fail because now there's an `__init__(self):` which seems to be incompatible with the previous definition.

I'll add a test.